### PR TITLE
feat: Add new members to team page

### PR DESCRIPTION
**Pull request by 0xcirxle**
This pull request includes the following updates to the teams page:

- Added profiles for team members
- Updated the socials link for each member

### CODE REVIEW

File: constants/alums.ts [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
     Male = 'male',
     Female = 'female',
 }
 
 export const Alums = [
     {
         name: 'Richa',
-        designation: 'Software Developer, JP Morgan & Chase',
+        designation_a: 'Software Developer, JP Morgan & Chase',
+        designation_b: '',
```
Comment (SamarendraGouda) [inline constants/alums.ts:10]: what's the purpose of having a second designation for alums? We can remove it ig. then handle on the card component using a v-if

File: components/Card/Team.vue [hunk 2]
```diff
@@ -1,28 +1,36 @@
...
-        designation: string;
-        github: string;
-        twitter: string;
-        linkedin: string;
-        gender: string;
-    }>();
+const props = defineProps<{
+  name: string;
+  designation_a: string;
+  designation_b: string;
```
Comment (SamarendraGouda) [inline components/Card/Team.vue:30]: make the designation_b prop optional

File: components/Card/Team.vue [hunk 3]
```diff
@@ -1,28 +1,36 @@
...
+  github: string;
+  twitter: string;
+  linkedin: string;
+  gender: string;
+}>();
 </script>
```
Comment (SamarendraGouda) [inline components/Card/Team.vue:36]: remove EOL issue

File: constants/team.ts [hunk 4]
```diff
@@ -6,74 +6,372 @@ enum Gender{
...
+        name: 'Shradhdha Agrawal',
+        designation_a: 'Developer',
+        designation_b: '',
+        image: '',
+        github: 'https://github.com/practicalPanda7325/',
+        twitter: 'https://x.com/0xPanda7325/',
+        linkedin: 'https://www.linkedin.com/in/shradhdha-agrawal-44039128b/',
+        gender: Gender.Female
+    },
 ]
```
Comment (SamarendraGouda) [inline constants/team.ts:377]: EOL

File: components/Card/Team.vue [hunk 5]
```diff
@@ -1,28 +1,36 @@
...
-        </div>
+  <div class="flex flex-row gap-4 w-80">
+    <div class="w-24 h-24 bg-gray-100">
+      <SvgoProfileMale v-if="gender === 'male'" class="w-24 h-24 text-gray-400"/>
+      <SvgoProfileFemale v-else class="w-24 h-24 text-gray-400"/>
     </div>
+    <div class="flex flex-col gap-2 w-40">
+      <h3 class="text-base font-semibold">{{name}}</h3>
+      <p class="text-sm font-medium text-gray-500 break-words">{{designation_a}}</p>
+      <p class="text-sm font-medium text-gray-500 break-words">{{designation_b}}</p>
```
Comment (SamarendraGouda) [inline components/Card/Team.vue:10]: use a `v-if="!!designation_b"` to conditionally render

**Comment by 0xcirxle** [comment]
Resolved the issues with previous commit

**Review by SamarendraGouda — APPROVED** [review]

**Review by 0xpanicError — APPROVED** [review]

File: components/Card/Team.vue [hunk 6]
```diff
@@ -1,28 +1,38 @@
...
-            <p class="flex flex-row items-center justify-start gap-2 text-xs font-medium text-gray-400">
-                <SvgoGithub class="w-4 h-4 cursor-pointer hover:text-gray-600" />
-                <SvgoTwitter class="w-4 h-4 cursor-pointer hover:text-gray-600" />
-                <SvgoLinkedin class="w-4 h-4 cursor-pointer hover:text-gray-600" />
-            </p>
-        </div>
+  <div class="flex flex-row gap-4 w-80">
+    <div class="w-24 h-24 bg-gray-100 self-stretch">
+      <img v-if="image" :src="image" :alt="name" class="w-24 h-full object-cover" />
+      <SvgoProfileMale v-else-if="props.gender === 'male'" class="w-24 h-24 text-gray-400"/>
```
Comment (SamarendraGouda) [inline components/Card/Team.vue:5]: `props.gender` --> `gender`

**Review by SamarendraGouda — APPROVED** [review]

**Pull request merged**